lib/neighborhood.ex

```diff
@@ -0,0 +1,21 @@
+defmodule GameOfLife.Neighborhood do
+
+  def count_live_neighbors_of(cells) do 
```

Comment:
The necessity of this is not immediately clear. Can you please explain what the purpose of this method is?